network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -351,9 +351,6 @@ def get_context(self, request, *args, **kwargs):
         context['current_category'] = None
         context['products'] = products
         context['web_monetization_pointer'] = settings.WEB_MONETIZATION_POINTER
-        pni_home_page = BuyersGuidePage.objects.first()
```

Comment:
Removing the logic that was used in order to pass the home page into the context